dlab_core/infrastructure/repositories.py

```diff
@@ -237,15 +385,26 @@ def _load_data(self):
 
 
 class SQLiteRepository(BaseFileRepository):
+    """Repository based on file data.
+
+    :type location: str
+    :param location: Data source file location.
+
+    :type table_name: str
+    :param table_name: Data source table name.
+
+    :type key_field_name: str
+    :param key_field_name: UUID field name.
+    """
 
     ALL_QUERY_TEMPLATE = 'SELECT {key}, {value} FROM {table}'
     ONE_QUERY_TEMPLATE = ALL_QUERY_TEMPLATE + ' where {key}=?'
 
     LC_READING_ERROR = 'Error while data reading with message "{msg}".'
 
-    def __init__(self, absolute_path, table_name, key_field_name='key',
+    def __init__(self, location, table_name, key_field_name='key',
                  value_field_name='value'):
-        super(SQLiteRepository, self).__init__(absolute_path)
+        super(SQLiteRepository, self).__init__(location)
 
         # TODO: add validation table, key and value needs to be string
         self._table_name = table_name
```

Comment:
line 416
settings = dict(...) -> {...}
https://medium.com/@jodylecompte/dict-vs-in-python-whats-the-big-deal-anyway-73e251df8398